claroty/info.json

```diff
@@ -5,7 +5,7 @@
   "publisher": "Fortinet",
   "cs_approved": true,
   "cs_compatible": true,
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
did you remove the old doc link?

Reply:
not removed for now, but will add new one once doc link is created